src/utils/ucc_mpool.c

```diff
@@ -57,6 +57,9 @@ ucc_status_t ucc_mpool_init(ucc_mpool_t *mp, size_t priv_size, size_t elem_size,
     ucs_ops->chunk_release = ucc_mpool_chunk_release_wrapper;
     ucs_ops->obj_init      = NULL;
     ucs_ops->obj_cleanup   = NULL;
+#if HAVE_UCS_MPOOL_OPS_T_OBJ_STR
+    ucs_ops->obj_str       = NULL;
```

Comment:
maybe instead do calloc for ucs_ops or memset 0 to avoid conditional compilation?

Reply:
right, i can do that. i just thought to be explicit in case we want to define obj_str to smth (same way ucx uses it). don't really have a strong opinion. can switch to memset. both fixes are straight forward. @bureddy wdyt?

Reply:
lets use memset to have a cleaner code.  we can add config check if we have use case to set obj_str